src/tests/regress/data/sql/float8.sql

```diff
@@ -106,13 +106,13 @@ select floor(f1) as floor_f1 from float8_tbl f;
 SET extra_float_digits = 0;
 
 -- square root
---@ SELECT sqrt(double precision '64') AS eight;
+SELECT sqrt(double precision '64') AS eight;
 
---@ SELECT |/ double precision '64' AS eight;
+SELECT |/ double precision '64' AS eight;
 
---@ SELECT f.f1, |/f.f1 AS sqrt_f1
---@    FROM FLOAT8_TBL f
---@    WHERE f.f1 > '0.0';
+SELECT f.f1, |/f.f1 AS sqrt_f1
+   FROM FLOAT8_TBL f
+   WHERE f.f1 > '0.0';
```

Comment:
```
-          f1          |        sqrt_f1
-----------------------+-----------------------
-               1004.3 |      31.6906926399535
- 1.2345678901234e+200 | 1.11111110611109e+100
- 1.2345678901234e-200 | 1.11111110611109e-100
+          f1          |         sqrt_f1
+----------------------+-------------------------
+               1004.3 |      31.690692639953454
+ 1.2345678901234e+200 | 1.1111111061110856e+100
+ 1.2345678901234e-200 | 1.1111111061110855e-100
 (3 rows)
```

The failure here is due to PostgreSQL setting `extra_float_digits = 0` is ignored by RisingWave. This setting reduces the precision of textual output, is not recommend for general use, but used here to allow imprecision comparison.

Our take of such inability to match PostgreSQL is to keep this disabled in regress test and write our own copy as e2e slt tests, inside `e2e_test/batch/functions`. There you can write `abs(sqrt($input::double precision) - $expected) < 1e-15` to test equality allowing an error less than 1e-15.

Reply:
thanks I write an e2e test instead however `1e-15` is too small for the first case sqrt(1004.3) so I used `1e-12` instead.

I could not repro the values on `float8.out` file with my postgres12
```
postgres=# ALTER SYSTEM SET extra_float_digits TO 0;
ALTER SYSTEM
postgres=# SELECT sqrt('1.2345678901234e+200');
          sqrt
-------------------------
 1.1111111061110856e+100
(1 row)

postgres=# SELECT sqrt(1004.3);
        sqrt
--------------------
 31.690692639953454
(1 row)

postgres=# SELECT sqrt('1.2345678901234e-200');
          sqrt
-------------------------
 1.1111111061110855e-100
(1 row)

```
vs
`SELECT f.f1, |/f.f1 AS sqrt_f1
   FROM FLOAT8_TBL f
   WHERE f.f1 > '0.0';
          f1          |        sqrt_f1        
----------------------+-----------------------
               1004.3 |      31.6906926399535
 1.2345678901234e+200 | 1.11111110611109e+100
 1.2345678901234e-200 | 1.11111110611109e-100
(3 rows)`

Reply:
The `extra_float_digits` setting was not effective, because it is session level and should use `SET` rather than `ALTER SYSTEM SET`. You can also check its value via `show extra_float_digits;`

btw we can still enable the 2 simple cases above, right?

Reply:
thanks a lot I see, I also re-enabled the 2 simple cases.
```
postgres=# show extra_float_digits;
 extra_float_digits
--------------------
 1
(1 row)
```